Makefile

```diff
@@ -1,11 +1,76 @@
 include .bingo/Variables.mk
 
-.DEFAULT_GOAL=textile
+.DEFAULT_GOAL=install
 
-GOVVV_FLAGS=$(shell $(GOVVV) -version git -flags -pkg $(shell go list ./buildinfo))
+TXTL_BUILD_FLAGS?=CGO_ENABLED=0
+TXTL_VERSION?="none"
+GOVVV_FLAGS=$(shell $(GOVVV) -flags -version $(TXTL_VERSION) -pkg $(shell go list ./buildinfo))
 
-textile: $(GOVVV)
-	go install -ldflags="${GOVVV_FLAGS}" ./...
+build: $(GOVVV)
+	$(TXTL_BUILD_FLAGS) go build -ldflags="${GOVVV_FLAGS}" ./...
+.PHONY: build
+
+build-hub: $(GOVVV)
+	$(TXTL_BUILD_FLAGS) go build -ldflags="${GOVVV_FLAGS}" ./cmd/hub
+.PHONY: build-hub
+
+build-hubd: $(GOVVV)
+	$(TXTL_BUILD_FLAGS) go build -ldflags="${GOVVV_FLAGS}" ./cmd/hubd
+.PHONY: build-hubd
+
+build-buck: $(GOVVV)
+	$(TXTL_BUILD_FLAGS) go build -ldflags="${GOVVV_FLAGS}" ./cmd/buck
+.PHONY: build-buck
+
+build-buckd: $(GOVVV)
+	$(TXTL_BUILD_FLAGS) go build -ldflags="${GOVVV_FLAGS}" ./cmd/buckd
+.PHONY: build-buckd
+
+install: $(GOVVV)
+	$(TXTL_BUILD_FLAGS) go install -ldflags="${GOVVV_FLAGS}" ./...
+.PHONY: install
+
+install-hub: $(GOVVV)
+	$(TXTL_BUILD_FLAGS) go install -ldflags="${GOVVV_FLAGS}" ./cmd/hub
+.PHONY: install-hub
+
+install-hubd: $(GOVVV)
+	$(TXTL_BUILD_FLAGS) go install -ldflags="${GOVVV_FLAGS}" ./cmd/hubd
+.PHONY: install-hubd
+
+install-buck: $(GOVVV)
+	$(TXTL_BUILD_FLAGS) go install -ldflags="${GOVVV_FLAGS}" ./cmd/buck
+.PHONY: install-buck
+
+install-buckd: $(GOVVV)
+	$(TXTL_BUILD_FLAGS) go install -ldflags="${GOVVV_FLAGS}" ./cmd/buckd
+.PHONY: install-buckd
+
+define gen_release_files
+	$(GOX) -osarch=$(3) -output="build/$(2)/$(2)_${TXTL_VERSION}_{{.OS}}-{{.Arch}}/$(2)" -ldflags="${GOVVV_FLAGS}" $(1)
+	mkdir -p build/dist; \
+	cd build/$(2); \
+	for release in *; do \
+		cp ../../LICENSE ../../README.md $${release}/; \
```

Comment:
I think we can just delete this file... it's empty anyway

Reply:
Which file do you mean?

Reply:
Oops. README.md

Reply:
Oh, I'll delete the `dist/README.md` because it is empty. But this command is actually copying the repo's main `README.md` which is full of good information. Should we leave that?

Reply:
Ah, I see... yeah nice let's keep that